projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -1,10 +1,19 @@
-<cx-carousel
-  [items]="items$ | async"
-  [title]="title$ | async"
-  [template]="carouselItem"
-  itemWidth="285px"
+<ng-container
+  *ngIf="merchandisingCarouselModel$ | async as merchandisingCarouselModel"
 >
-</cx-carousel>
+  <div
+    class="data-cx-merchandising-carousel"
+    [cxAttributes]="merchandisingCarouselModel.metadata"
+    [cxAttributesNamePrefix]="'data-cx-merchandising-carousel'"
+  ></div>
```

Comment:
I just noticed: does this `div` need to wrap `cx-carousel`?

Reply:
No, with it here it ends up inside the top level `cx-merchandising-carousel` as a sibling to `cx-carousel` which is good enough for our needs